optimum/neuron/pipelines/transformers/sentence_transformers.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict
+
+from transformers.pipelines.base import GenericTensor, Pipeline
+
+from optimum.utils import is_sentence_transformers_available
+
+
+def is_sentence_transformer_model(model: str, token: str = None, revision: str = None):
+    """Checks if the model is a sentence transformer model based on provided model id"""
+    if is_sentence_transformers_available():
+        from optimum.exporters.tasks import TasksManager
```

Comment:
We could put the import at the top of the file.

Reply:
done